### ISSUE TITLE
After logging into Firefox account on brand new device, SimpleLogin extension automatically logs in too

### ISSUE THREAD

**Issue by Asheq** [simple-login/browser-extension#121]
I noticed that if I login to my Firefox account on a brand new device (or brand new Firefox profile), the SimpleLogin extension automatically logs in to SimpleLogin too. 

I understand this is convenient, but I wonder if it is a security risk?

If someone manages to compromise my Firefox account, then they also automatically get access to SimpleLogin via the extension.

**Comment by FozzieHi** [comment]
I agree, I think this API keys should be saved in local storage. I think this is currently happening because [all settings are saved in sync storage](https://github.com/simple-login/browser-extension/blob/1f4bff5c7c922cdd82d69e2bb8c69a67d722d8eb/src/popup/SLStorage.js). It should be fairly simple to move just the API values to local storage, but this may require users to re-input their API keys. What do you think about this @nguyenkims?
